src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react'
 import { Control, Controller, DeepMap, FieldError, UseFormMethods } from 'react-hook-form'
-import { isValid } from 'date-fns'
 import { useCategories } from '@/api/hooks'
+import { useDrafts } from '@/hooks'
 import { Checkbox, Datepicker, FormField, RadioButton, Select, SelectItem, TextArea } from '@/shared/components'
-import { requiredValidation } from '@/utils/formValidationOptions'
+import { requiredValidation, dateValidation } from '@/utils/formValidationOptions'
 import { languages } from '@/config/languages'
-import { FormContainer, StyledHeaderTextField, StyledRadioContainer } from './EditVideoForm.style'
+import {
+  FormContainer,
+  StyledHeaderTextField,
+  StyledRadioContainer,
+  DeleteVideoContainer,
+  DeleteVideoButton,
+} from './EditVideoForm.style'
 
 const visibilityOptions: SelectItem[] = [
   { name: 'Public (Anyone can see this video)', value: 'public' },
```

Comment:
This could use `true/false` as values, the same as we do in channel edit. Then we don't need to do comparison when saving/loading the draft